main/meleeMenu.c

```diff
@@ -113,7 +124,7 @@ void deinitMeleeMenu(meleeMenu_t* menu)
 int addRowToMeleeMenu(meleeMenu_t* menu, const char* label)
 {
     // Make sure there's space for this row
-    if(menu->numRows < MAX_ROWS)
+    if(menu->numRows < MAX_ROWS - 1)
```

Comment:
Why did this get a `- 1`?

Reply:
compiler deemed the `if` statement to always be true otherwise due to data type limitations. I'm just now realizing `numRows` is not a 0-indexed number, so `MAX_ROWS` will need to be 255 instead.